config/jobs/kubernetes/kops/kops-presubmits-network-plugins.yaml

```diff
@@ -79,7 +79,7 @@ presubmits:
         - /workspace/scenarios/kubernetes_e2e.py
         args:
         - --aws
-        - --aws-cluster-domain=test-cncf-aws.k8s.io
+        - --aws-cluster-domain=test-cncf-calico.k8s.io
```

Comment:
are you sure this is safe? i don't know exactly how the AWS accounts are setup but I would imagine there is a hosted zone for test-cncf-aws.k8s.io and changing this might mean that kops can't find a hosted zone to add records to.

what exactly is the problem caused by having them all use the same aws-cluster-domain?

Reply:
I was thinking they may clash if they run at same time, like the ARM64 ones last week.

Reply:
You may be right, so I changed back.